app/models/admin/action_log.rb

```diff
@@ -35,6 +35,11 @@ def set_changes
       self.recorded_changes = target.attributes
     when :update, :promote, :demote
       self.recorded_changes = target.previous_changes
+    when :change_email
+      self.recorded_changes = ActiveSupport::HashWithIndifferentAccess.new(
+        email: [target.email, nil],
+        unconfirmed_email: [nil, target.unconfirmed_email]
+      )
```

Comment:
This might be a bit contentious, but essentially in the ActionLog view, you want to see both the current email and what it's been changed to. This isn't captured with `target.previous_changes` as we're not updating the email directly, but rather requiring confirmation of the email update.